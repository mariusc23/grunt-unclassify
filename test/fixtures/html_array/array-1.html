<!DOCTYPE html>
<html lang="en">
  <body>
    <div class="only"></div>
  </body>
</html>
