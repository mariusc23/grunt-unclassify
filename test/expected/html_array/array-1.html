<!DOCTYPE html>
<html lang="en">
  <body>
    <div></div>
  </body>
</html>
